cpp/include/kvikio/cufile_config.hpp

```diff
@@ -22,12 +22,12 @@
 namespace kvikio {
 namespace detail {
 
-[[nodiscard]] inline const std::string lookup_config_path()
+[[nodiscard]] inline const char* lookup_config_path()
 {
   const char* env = std::getenv("CUFILE_ENV_PATH_JSON");
   if (env != nullptr && std::filesystem::exists(env)) { return env; }
   if (std::filesystem::exists("/etc/cufile.json")) { return "/etc/cufile.json"; }
```

Comment:
side question: could this be done once (e.g. init a local static var)?

Reply:
It is done once, `config_path()` calls it to assign a static variable

Reply:
Got it. I assumed repeated calls because of the optimization.